phoenix-core/src/main/antlr3/PhoenixSQL.g

```diff
@@ -967,9 +967,14 @@ NUMBER
     :   POSINTEGER
     ;
 
-// Exponential format is not supported.
 DECIMAL
```

Comment:
All of those will work, and we capture 'e' and 'E' as well.  The only one that wont work is "+1.5e-7" (listed in the description of https://issues.apache.org/jira/browse/PHOENIX-1814 as an example value).  I tested inserting regular numbers (non-exponential notation) with a '+' before them and the value doesn't parse at all.